packages/component-library/src/CivicStoryCard/CivicStoryLink.js

```diff
@@ -4,26 +4,41 @@ import { Link } from "react-router";
 import { jsx, css } from "@emotion/core";
 
 const storyLinkClass = css`
-  & a {
+  & a, button {
     display: flex;
     flex-wrap: nowrap;
     align-items: center;
     justify-content: space-between;
     padding: 8px 12px;
     border-bottom: none;
     color: #000;
+    border: none;
+    background-color: #eee;
+    &:focus {
+      outline: 0;
+    }
```

Comment:
If we're going to set outline to 0 here, we should have some alternative styles for focus -- read more here for why: http://outlinenone.com

#823 is a related open issue but needs a design. For now we could remove this reset and leave the default focus.

Reply:
Thanks for the feedback! My latest commit removed the focus styling.